src/integrationTest/resources/tests/dailylist/DailyListServiceTest/insert1OkJsonAndXml/expectedResponse2.json

```diff
@@ -20,5 +20,7 @@
   "contentObjectId": null,
   "clipId": null,
   "externalLocation": null,
-  "externalLocationTypeEntity": null
+  "externalLocationTypeEntity": null,
+  "subcontentObjectId": null,
+  "subcontentPosition": null
```

Comment:
Should this be in snake_case to follow HMCTs standards if they are new fields?

Reply:
See response to similar comment.